nixos/modules/virtualisation/virtualbox-host.nix

```diff
@@ -102,7 +102,8 @@ in
       "VBoxNetNAT"
       "VBoxSDL"
       "VBoxVolInfo"
-      "VirtualBox"
+      "VirtualBoxVM"
+      "VBoxHeadless"
```

Comment:
This appears to be a repetition of the first line in this list.
```suggestion
```

Reply:
Thanks for pointing out. Fixed